### ISSUE TITLE
ANSI codes get injected when using Promise.all with dynamic imports.

### ISSUE THREAD

**Issue by Pomax** [rollup/rollup#3839]
- Rollup Version: 2.32.1
- Operating System (or Browser): Win10 x64
- Node Version (if applicable): 14.10.0
- Link to reproduction (IMPORTANT, read below):

>  Issues without minimal reproductions will be closed! Please provide a link to one by:
>  1. Using the REPL at https://rollupjs.org/repl/, or
>  2. Using the REPL.it reproduction template at https://repl.it/@rollup/rollup-repro
>     (allows full use of all rollup options and plugins), or
>  3. Provide a minimal repository link (Read https://git.io/fNzHA for instructions).
>     These may take more time to triage than the other options.

This problem cannot be reproduced online because the output gets rewritten to something that's clean enough to show in a browser.

I've set up a branch `rollup-up` on https://github.com/Pomax/Font.js with node_modules checked in (as it's a pure rollup bug). Clone the repo, `git checkout rollup-bug`, then `npm test`, then load result.js in a code editor.

### Expected Behavior

Something normal?

### Actual Behavior

```js

[36m[1m//→ input.js:[22m[39m
var input = Promise.all([ import('./import-525a1774.js') ]);

export default input;

[36m[1m//→ import-525a1774.js:[22m[39m
class X {};

export default X;
```
### Code snippets

The branch is basically almost no code at all, it's two one-liner files and the rollup command.

`input.js`:

```js
export default Promise.all([ import("./import.js") ])
```

`import.js`:

```
export default class X {};
```

Then roll this up using `rollup --format=esm input.js > result.js`

**Comment by lukastaegert** [comment]
You can get rid of the codes by setting the `NO_COLOR=1` environment variable. This should not happen, though, as Rollup sets the flag internally automatically when it determines it is running in a pipe.

So on my machine when I used the `>` operator to write the output to the file, no ANSI codes were added. I wonder what is different with your setup.

**Comment by Pomax** [comment]
I'm seeing this on windows 10 pro x64 when running using the standard `cmd` shell.

**Comment by Pomax** [comment]
Note that this is still happening.

**Comment by Pomax** [comment]
Still seeing this still happen on v2.53.0 even with `NO-COLOR=` set through cross-env. It would be _super_ useful if there was an actual --no-color flag I could pass to rollup

**Comment by dnalborczyk** [comment]
@Pomax you should be able to set the `NO_COLOR` env variable (notice the underscore).

**Comment by Pomax** [comment]
Good catch, although having to set an env. var at all is still very cross-platform-problematic.
